### PR TITLE
Translation and Documentation Pinning

### PR DESCRIPTION
**Pull request by Daltz333**
Details the process used when tagging and releasing documentation/translations.

### CODE REVIEW

File: docs/release_processes/translation-documentation-pinning.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Consistency
+- Ease of Translation
+
+Translations are accessible via the language code. IE: [https://docs.wpilib.org/es/latest/](https://docs.wpilib.org/es/latest/).
+
+## Our Approach
+
+Translations happen consistently throughout the year, as long as there is documentation momentum. To make it easy to understand, we break this process up into two parts.
+
+Reference: ``/latest/`` is the "bleeding edge" branch of the docs. Accessible from [https://docs.wpilib.org/en/latest/](https://docs.wpilib.org/en/latest/) and ``/stable/`` is the "year-to-year" version of the docs. This is accessible from [https://docs.wpilib.org/en/stable/](https://docs.wpilib.org/en/stable/)
```
Comment (AustinShalit) [inline docs/release_processes/translation-documentation-pinning.md]:
```suggestion
Reference: ``/latest/`` is the "leading edge" branch of the docs. Accessible from [https://docs.wpilib.org/en/latest/](https://docs.wpilib.org/en/latest/) and ``/stable/`` is the "year-to-year" version of the docs. This is accessible from [https://docs.wpilib.org/en/stable/](https://docs.wpilib.org/en/stable/)
```

File: docs/release_processes/translation-documentation-pinning.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Beginning of Season:
+- Translation and Documentation ``/latest/`` and ``/stable`` are at an equal point in history. This allows us to make "bleeding" edge changes to documentation and translations. The benefit of this, is making it immediately live on both versions of the site.
```
Comment (AustinShalit) [inline docs/release_processes/translation-documentation-pinning.md]:
```suggestion
- Translation and Documentation ``/latest/`` and ``/stable`` are at an equal point in history. This allows us to make "leading" edge changes to documentation and translations. The benefit of this, is making it immediately live on both versions of the site.
```

File: docs/release_processes/translation-documentation-pinning.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+End of Season:
+- Translations & Documentation have ``stable`` locked for new **content** additions but are able to be translated. ``latest`` is unable to be translated at this time. 
+- ``latest`` for the [documentation repo](https://github.com/wpilibsuite/frc-docs) is tagged at a yearly date for archival purposes (IE: 2022).
+
+Mid-Summer:
+- Translations have ``stable`` locked for new **content** and new **translations**. ``latest`` is now available for translation. Translation repo is tagged at a yearly date (IE: 2022).
+
+This release cycle ensures that our visitors always get the most relevant and stable documentation. We don't want visitors receiving development changes to documentation for things they are still working on for the current year.
+
+Last Updated: 6/25/2021
```
Comment (AustinShalit) [inline docs/release_processes/translation-documentation-pinning.md]:
```suggestion
```

Git history handles this.

File: docs/release_processes/translation-documentation-pinning.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Documentation & Translation Pinning
```
Comment (AustinShalit) [inline docs/release_processes/translation-documentation-pinning.md]:
```suggestion
# Documentation
```

What do you think of just calling this documentation?

**Review by AustinShalit — CHANGES_REQUESTED** [review]

**Review by FeliciaAKelley — APPROVED** [review]

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**